lib/Stickersheet.js

```diff
@@ -0,0 +1,126 @@
+var Document = require('html-document/lib/Document').Document;
+var _ = require('lodash');
+
+var Stickersheet = module.exports = function(trixel) {
+    
+    // This stickersheet scales based on dpi. Normally the dpi would be 600
+    // Though if the page isn't 8.5 / 11 inches, things need to be programmatically repositioned.
+    
+    
+    // Our fake HTML document
+    var document = new Document();
+    
+    var pageWidthInches = 8.5; // Assumed to be inches
+    var pageHeightInches = 11; // Assumed to be inches
+    var dpi = 100; // Change to 600 for production or whatever the passed in dpi is (Passing in dpi not set up yet)
+    var pageWidth = pageWidthInches * dpi;
+    var pageHeight = pageHeightInches * dpi;
+    
+    var svgContainer = document.createElement("svg");
+    svgContainer.setAttribute("xmlns", "http://www.w3.org/2000/svg");
+    svgContainer.setAttribute("version", "1.1");
+    svgContainer.setAttribute("style", "width: "+pageWidth+"px; height: "+pageHeight+"px;");
+    
+    var trixels = {
+        t1: {},
+        t2: {},
+        t3: {},
+        t4: {},
+        t5: {},
+        t6: {}
+    }
+    
+    // Make it easier to grab values from the original trixel's viewBox
+    var viewBoxTmp = trixel.element.getAttribute("viewBox");
+    viewBoxTmp = viewBoxTmp.split(' ');
+    var trixelViewBox = {
+        x: viewBoxTmp[0],
+        y: viewBoxTmp[1],
+        width: viewBoxTmp[2],
+        height: viewBoxTmp[3]
+    }
+
+    var trixelWHratio = trixelViewBox.width / trixelViewBox.height;
+    
+    var trixelWidth = pageWidth * .3622; // Best guess based on mockup
+    var trixelHeight = trixelWidth / trixelWHratio;
+
+    function dupTrixel() {
+        // lodash's _.clone() method returns empty object if given a DOM node,
+        // even though these fake DOM nodes are of type Object so we do this.
+        var dupTrixel = document.createElement("svg");
+        dupTrixel.innerHTML = trixel.element.innerHTML
+        dupTrixel.setAttribute("viewBox", trixel.element.getAttribute("viewBox"));
+        dupTrixel.setAttribute("width", trixelWidth);
+        return dupTrixel;
+    }
+    
+    Object.keys(trixels).map(function(trixel) {
+        trixels[trixel] = dupTrixel()
```

Comment:
Maybe rename `trixel` here.  `trixel` is currently being used in the upper scope to represent the "master copy" of the trixel for this sticker sheet.